src/test/java/messenger/InputOutputTestSuite.java

```diff
@@ -0,0 +1,145 @@
+package messenger;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.io.TempDir;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+
+/**
+ * @author Olga Petrova
+ */
+
+@ExtendWith(MockitoExtension.class)
+public class InputOutputTestSuite {
```

Comment:
it is integration test in case of this application. it should be shown in its name.

Reply:
fixed, added new class for integration tests